translations/base-fr.yaml

```diff
@@ -475,21 +476,21 @@ buildings:
     cutter:
         default:
             name: &cutter Découpeur
-            description: Coupe une forme de haut en bas et sort les deux parties. <strong>Si vous n'utilisez qu'une seule partie, assurez-vous de détruire l'autre ou sinon, gare au blocage !</strong>
+            description: Coupe une forme de haut en bas et sort les deux parties. <strong>Si vous n’utilisez qu’une seule partie, assurez-vous de détruite l’autre ou sinon, gare au blocage !</strong>
```

Comment:
typo got reintroduced here

Reply:
Typo's still there

Reply:
The curly apostrophe is so cute !!!

Reply:
For example:
![Comparison between the two apostrophe in three fonts](https://zupimages.net/up/20/37/894o.png)

Reply:
I think you meant to reply to the comment above this. This one is about the misspelling in "détruite".

Re: `’`, yes I blame the character, not the appearance of the glyph.

Part of it is that some misbehaving programs assume that all single quotes should be curly quotes, even when they aren't, and mangle e.g. code. That's not the fault of the poor innocent character, but it makes its existence harmful overall.

The part that *is* the character's fault is that it doesn't make sense: it's both a closing single quote, and an apostrophe — both jobs already done by `'`, which (1) makes more sense since it's also an opening single quote and doesn't have a strange asymmetrical meaning (2) is much more convenient because it's in ASCII. It makes sense to split opening quote/closing quote/apostrophe, or to unify them into a single character, but splitting this way is just confusing.